loafwallet/src/ModalPresenter.swift

```diff
@@ -899,7 +903,7 @@ class ModalPresenter : Subscriber, Trackable {
         guard notReachableAlert == nil else { return }
         let alert = InAppAlert(message: S.LitewalletAlert.noInternet, image: #imageLiteral(resourceName: "BrokenCloud"))
         notReachableAlert = alert
-        let window = UIApplication.shared.keyWindow!
+        let window = UIApplication.shared.windows.filter{$0.isKeyWindow}.first!
```

Comment:
@kcw-grunt is it safe?! is `windows array ` always contains at least on window of type `isKeyWindow` ?

Reply:
Well the forced unwrap is even more dangerous.  Let me guard it in all cases

Reply:
even better!